### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by SmashedFrenzy16**
Update `README.rst` with a table instead of bullet points for the dependency versions.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -128,13 +128,21 @@ own rotozoom function.  The surfarray module requires the Python
...
++----------+------------------------+
+| SDL_mixer| >= 2.0.0               |
++----------+------------------------+
+| SDL_image| >= 2.0.2               |
++----------+------------------------+
+| SDL_ttf  | >= 2.0.11              |
++----------+------------------------+
+| SDL_gfx  | (Optional, vendored in)|
++----------+------------------------+
+| NumPy    | >= 2.6.2 (Optional)    |
```
Comment (MyreMylar) [inline README.rst:144]: You have changed the version number here for NumPy.

Comment (SmashedFrenzy16) [reply README.rst]: @MyreMylar I have fixed this.

**Comment by SmashedFrenzy16** [comment]
@MyreMylar I have fixed the `numpy` issue.

**Review by MyreMylar — APPROVED** [review]
OK, I merged you up to main and this looks good to me now.

I prefer the aesthetics of a table here. 👍

**Review by Starbuck5 — APPROVED** [review]
Sure

**Pull request merged**